src/geometry/manifold/ManifoldGeometry.cc

```diff
@@ -1,19 +1,19 @@
 // Portions of this file are Copyright 2023 Google LLC, and licensed under GPL2+. See COPYING.
-#include "ManifoldGeometry.h"
-#include "Polygon2d.h"
+#include "geometry/manifold/ManifoldGeometry.h"
+#include "geometry/Polygon2d.h"
 #include "manifold/cross_section.h"
 #include "manifold/manifold.h"
-#include "PolySet.h"
-#include "PolySetBuilder.h"
-#include "PolySetUtils.h"
-#include "manifoldutils.h"
-#include "ColorMap.h"
+#include "geometry/PolySet.h"
+#include "geometry/PolySetBuilder.h"
+#include "geometry/PolySetUtils.h"
+#include "geometry/manifold/manifoldutils.h"
+#include "glview/ColorMap.h"
 #include "src/glview/RenderSettings.h"
```

Comment:
Just glview/RenderSettings.h ?

Reply:
ah, that also got missed by the script that wasn't expecting that prefix.